releng/jenkins/check-versions/Jenkinsfile

```diff
@@ -243,5 +257,9 @@ def getXtextBootstrapVersion (branch) {
  */
 
 def getVersionFromBOM (id, branch='master') {
-  return sh (script: "curl -sS https://raw.githubusercontent.com/eclipse/xtext-lib/${branch}/org.eclipse.xtext.dev-bom/build.gradle |grep -Po 'api \\\"${id}:\\K[^\"]*'", returnStdout: true).trim()
+  env.BRANCH_REF="${branch}"
+  sh (script: 'curl -sSL -H "Cookie: logged_in=no" -H "Authorization: token $GITHUB_API_TOKEN" -H "Content-Type: text/plain; charset=utf-8" https://api.github.com/repos/eclipse/xtext-lib/contents/org.eclipse.xtext.dev-bom/build.gradle?ref=$BRANCH_REF | jq -r ".content" | base64 -d > av.txt')
```

Comment:
i dont get this running without using a file.
either the interpolation wont work or the grep

Reply:
i think using a file is not that bad. even less strain on the rate limit